### PR TITLE
fix: toast invalid hook issue

### PR DESCRIPTION
**Pull request by Harris-YWang**
## Description

Because the incompatibility between React.memo and Symphony's project and `<ToastContainer />` has been fully wrapped by `react-toastify` like a black-box, we think we don't have to wrap it again in adslot-ui for the moment.

Also, `react-toastify` use `toast()` to trigger a toast adding to the `<ToastContainer />`. Initially we wrapped it into a component, which should works well in JSX files. However, with Symphony's project, they are gonna use this function in the redux action, which is unclear for people to manage a component in the action.js file. So, adding a `Toast.renderNotification()` function, exactly has the same UI display as `<Toast.Notificaiton />`, but we can use it as a function rather than a functional component.

btw, fix small bugs of the doc.

## Does this PR introduce a breaking change?

- [ ] Yes
- [x] No

## Manual testing step?

Manual tested by @devanshiAdslot

### CODE REVIEW

File: src/components/Toast/ToastContainer.jsx [hunk 1]
```diff
@@ -39,4 +39,4 @@ ToastContainerComponent.defaultProps = {
   pauseOnHover: true,
 };
 
-export default React.memo(ToastContainerComponent);
```
Comment (lightbringer1991) [inline src/components/Toast/ToastContainer.jsx:42]: yea I think we shouldn't wrap anything in `adslot-ui`, users can do it themselves.

File: src/components/Toast/ToastNotification.jsx [hunk 2]
```diff
@@ -1,29 +1,43 @@
...
+  const toastClass = getToastClass(theme);
+  const toastMessage = <ToastMessage toastClass={toastClass} title={title} message={message} />;
+  const toastOptions = options(props);
 
-  toast(toastMessage, options);
+  toast.info(toastMessage, toastOptions);
   return null;
 };
 
+export const renderNotification = ({ theme, title, message, ...props }) => {
```
Comment (knilink) [inline src/components/Toast/ToastNotification.jsx:34]: maybe use a different name, such as just `info`?
as it's not a typical render function

Comment (Harris-YWang) [reply src/components/Toast/ToastNotification.jsx:34]: how about something like `Toast.notify`?

**Comment by superpower1** [comment]
Please don't commit your build files.

**Comment by Harris-YWang** [comment]
> Please don't commit your build files.

Solved

**Pull request merged**